sublime-package.json

```diff
@@ -315,15 +315,20 @@
               "default": false,
               "markdownDescription": "Show symbol action links in hover popup if available."
             },
-            "inhibit_explicit_completions": {
+            "only_show_lsp_completions": {
               "type": "boolean",
               "default": false,
-              "markdownDescription": "Disable Sublime Text's snippet completions."
+              "deprecationMessage": "This setting is deprecated. Use a combination of \"inhibit_snippet_completions\" and \"inhibit_word_completions\" instead."
             },
-            "inhibit_word_completions": {
+            "inhibit_snippet_completions": {
               "type": "boolean",
               "default": false,
-              "markdownDescription": "Disable Sublime Text's word completions. When set to `true`, this also disables Sublime Text's internal completion sorting algorithm and instead uses the sorting defined by the relevant language server."
+              "markdownDescription": "Disable Sublime Text's (and your own) snippet completions. Note that your language server may still provide completion items of the \"snippet\" kind."
+            },
+            "inhibit_word_completions": {
+              "type": "boolean",
+              "default": true,
+              "markdownDescription": "Disable Sublime Text's word completions. \"word\" completion means Sublime Text's internal completer that takes words from the current buffer you're editing and presents them in the auto-complete widget. When set to `true`, this also disables Sublime Text's internal completion sorting algorithm and instead uses the sorting defined by the relevant language server."
```

Comment:
> When set to `true`, this also disables Sublime Text's internal completion sorting algorithm

Now, this seems to happen irrespective of those options.

Reply:
INHIBIT_REORDER doesn't seem to really do anything if word completions are enabled. ST seems to prefer word completions over plugin completions. So, I'd say just enable it when there's at least one item.